File: 단계별로 풀어보기/조건문/6단계/오븐시계.cpp
#include <iostream>
using namespace std;

int main(){
  int hour, minute, time;
  cin>>hour>>minute>>time;
  
  hour+=time/60;
  minute+=time%60;
  
  if(minute>=60){
    minute-=60;
    hour+=1;
  }
  if(hour>=24){
    hour-=24;
  }
  
  cout<<hour<<' '<<minute<<endl;
  
  return 0;
}

File: 단계별로 풀어보기/조건문/6단계/오븐 시계.py
def timer(): //변수 계산 순서 중요
    hour, minute=map(int,input().split(' '))
    time=int(input())
    
    minute+=time%60
    if(minute>=60):
        hour+=1
        minute-=60
    
    hour+=time//60
    if(hour>=24):
        hour-=24
    
    print(hour, minute)
    
if __name__=='__main__':
    timer()
